ufpy/github/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .download import file as download_file
+from .download import folder as download_folder
+from .download import repo as download_repo
```

Comment:
Fixes many errors with packaging
```suggestion
from ufpy.github.download import file as download_file
from ufpy.github.download import folder as download_folder
from ufpy.github.download import repo as download_repo
```